### ISSUE TITLE
cargo creates multiple web-view

### ISSUE THREAD

**Issue by xasopheno** [Boscop/web-view#161]
When running with `cargo watch --exec "run" and an actix backend server running in separate thread, the cargo watch reloads leave the web-view hanging. Multiple webviews end up being created on subsequent runs and the system quickly becomes unusable. Any ideas?

**Comment by zxey** [comment]
Which platform are you running on?

**Comment by xasopheno** [comment]
OSX. Would you like to see the code?

**Comment by zxey** [comment]
Yes, if you'd be so kind, thank you. I just tried to reproduce this, but was not able to.
